gradle.properties

```diff
@@ -1,10 +1,13 @@
 
 # Dependency versions
-kotlin_version = 1.2.61
-mustache_version = 0.9.5
-okhttp_version = 3.11.0
-gson_version = 2.8.5
-guava_version = 26.0-jre
+kotlin_version = 1.3.50
+mustache_version = 0.9.6
+okhttp_version = 3.14.4
+# latest: okhttp_version = 4.2.2
+gson_version = 2.8.6
+guava_version = 28.1-jre
 kotlintest_version = 2.0.7
-system_rules_version = 1.17.1
-truth_version = 0.42
\ No newline at end of file
+# latest:  kotlintest_version = 3.4.2 (https://github.com/kotlintest/kotlintest/releases vs
```

Comment:
looks like the repo this code is going to only has 2.0.7 (not the latest (3.4.2))